src/App/adapter/translate.js

```diff
@@ -1,18 +1,28 @@
 import { createInclude, createWhere } from './lib/create'
 import { stripEmptyKeys } from './lib/helpers'
-import { mergeState, parseState } from './lib/state'
+import { createPagination, mergeState, parseState } from './lib/state'
+
+const DEFAULT_CONFIG = {
+  endpoint: 'documents',
+  include: [],
+  pageSize: 5, // `false` to disable limit/pagination
+  page: 1,
+}
+
+function translate(diffState, initialState = [], queryConfig = {}) {
+  const config = { ...DEFAULT_CONFIG, ...queryConfig }
```

Comment:
Merging the custom query config with the default config. See examples in tests.

I'll put the `initialFilters` in the config object in my next PR.

Reply:
It feels more elegant to me to achieve this via mergeState

Reply:
Your decision though! Mine have a history of turning out "a bit silly".

Reply:
No worries, I'm hoping that my reasoning will make more sense once I show you some code.